build/commands/lib/config.js

```diff
@@ -455,83 +361,91 @@ Config.prototype.buildArgs = function () {
-    uphold_production_client_id: this.upholdProductionClientId,
-    uphold_production_client_secret: this.upholdProductionClientSecret,
-    uphold_production_fee_address: this.upholdProductionFeeAddress,
-    uphold_production_oauth_url: this.upholdProductionOauthUrl,
-    uphold_sandbox_api_url: this.upholdSandboxApiUrl,
-    uphold_sandbox_client_id: this.upholdSandboxClientId,
-    uphold_sandbox_client_secret: this.upholdSandboxClientSecret,
-    uphold_sandbox_fee_address: this.upholdSandboxFeeAddress,
-    uphold_sandbox_oauth_url: this.upholdSandboxOauthUrl,
-    zebpay_production_api_url: this.zebPayProductionApiUrl,
-    zebpay_production_client_id: this.zebPayProductionClientId,
-    zebpay_production_client_secret: this.zebPayProductionClientSecret,
-    zebpay_production_oauth_url: this.zebPayProductionOauthUrl,
-    zebpay_sandbox_api_url: this.zebPaySandboxApiUrl,
-    zebpay_sandbox_client_id: this.zebPaySandboxClientId,
-    zebpay_sandbox_client_secret: this.zebPaySandboxClientSecret,
-    zebpay_sandbox_oauth_url: this.zebPaySandboxOauthUrl,
     brave_version_major: versionParts[0],
     brave_version_minor: versionParts[1],
     brave_version_build: versionParts[2],
     chrome_version_string: this.chromeVersion,
-    brave_sync_endpoint: this.braveSyncEndpoint,
-    safebrowsing_api_endpoint: this.safeBrowsingApiEndpoint,
-    brave_variations_server_url: this.braveVariationsServerUrl,
-    updater_prod_endpoint: this.updaterProdEndpoint,
-    updater_dev_endpoint: this.updaterDevEndpoint,
-    webcompat_report_api_endpoint: this.webcompatReportApiEndpoint,
-    rewards_grant_dev_endpoint: this.rewardsGrantDevEndpoint,
-    rewards_grant_staging_endpoint: this.rewardsGrantStagingEndpoint,
-    rewards_grant_prod_endpoint: this.rewardsGrantProdEndpoint,
     brave_stats_api_key: this.braveStatsApiKey,
     brave_stats_updater_url: this.braveStatsUpdaterUrl,
     enable_hangout_services_extension: this.enable_hangout_services_extension,
     enable_cdm_host_verification: this.enableCDMHostVerification(),
     enable_pseudolocales: this.enable_pseudolocales,
     skip_signing: !this.shouldSign(),
-    sparkle_dsa_private_key_file: this.sparkleDSAPrivateKeyFile,
-    sparkle_eddsa_private_key: this.sparkleEdDSAPrivateKey,
-    sparkle_eddsa_public_key: this.sparkleEdDSAPublicKey,
     use_remoteexec: this.useRemoteExec,
     use_reclient: this.useRemoteExec,
     use_siso: false,
     use_libfuzzer: this.use_libfuzzer,
     enable_update_notifications: this.isOfficialBuild(),
-    brave_services_production_domain: this.braveServicesProductionDomain,
-    brave_services_staging_domain: this.braveServicesStagingDomain,
-    brave_services_dev_domain: this.braveServicesDevDomain,
-    brave_services_key_id: this.brave_services_key_id,
-    service_key_aichat: this.service_key_aichat,
     service_key_stt: this.service_key_stt,
     generate_about_credits: true,
   }
 
+  const forwardFromEnv = [
```

Comment:
this should probably be defined as part of Config up here https://github.com/brave/brave-core/pull/30262/files#diff-4a7ab71a64ac5712bb35fb91782847039cf2def6ee9c3d64a7efa66562a99acfR187 to keep this method a little smaller?

Reply:
and then `this.forwardFromEnv`

Reply:
I could go either way on this if we think it's better to keep everything for args defined in the same place, but this method is a bit cumbersome and would be nice to make it a bit smaller

Reply:
I found it a little cleaner to extract into static constant. What do you think @bridiver?
4d1d71ae05760537856269750192f4b18f391194

Reply:
My intention was to make clear that it relates to `Config`. I don't have a very strong preference and can also put into a `const` if you like.